src/index.html

```diff
@@ -4,10 +4,56 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__photo"
+        src="images/imac.jpeg"
+        alt="MAC"
+       >
+      <div class="card__description">
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">
+          Product code: 195434
+        </div>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__revievs-count">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__text">Price:</div>
+          <div class="card__number">$2,199</div>
+        </div>
+      </div>
+      <a
+       class="card__buy-button"
+       href="#BUY"
+       data-qa="hover"
+       >
```

Comment:
please, fix the formatting, the first and the last tag shoul be on the same distance (line)